### ISSUE TITLE
[QUESTION]

### ISSUE THREAD

**Issue by nosferatu500** [petyosi/react-virtuoso#456]
Hi, thanks for this library!

I have a question:

I saw that you added a new keyboard navigation feature. I am currently using react-virtuoso with react-beautiful-dnd. But I don't like how keyboard support works in react-beautiful-dnd and I want to use react-virtuoso for that. When I try to do this, I find that if I use the keyboard navigation to scroll through react-virtuoso, I cannot use the auto-scroll from react-beautiful-dnd.

Is it possible to keep the autoscrolling support from react-beautiful-dnd and use the keyboard support from react-virtuoso?

**Comment by nosferatu500** [comment]
I only need keyboard support for navigation. I don't need keyboard support for Drag&Drop. 
But also I would like to keep the ability to autoscroll, during Drag&Drop for the mouse.
